roles/mirror_ocp_release/tasks/fetch.yml

```diff
@@ -1,12 +1,4 @@
 ---
-- name: "Check if target file exists"
-  ansible.builtin.stat:
-    path: "{{ mor_dir }}/{{ mor_uri | basename }}"
-    get_checksum: false
-  register: target
-  when:
-    - not mor_force
```

Comment:
Since we're extracting the artifacts on a temporary directory, the file won't exist in advance.